.travis.yml

```diff
@@ -13,7 +16,8 @@ python:
   - 3.6
 
 script:
-    - set -e
-    - make -C docs html
-    - pip install doctr
-    - doctr deploy --built-docs docs/build/html --deploy-repo NSLS-II/NSLS-II.github.io suitcase
+  - set -e
+  - make -C docs html
+  - pip install doctr
```

Comment:
Any idea why not to put it to the `requirements-dev.txt`?

Reply:
We used to do so, but it's not really a requirement for local development. It's really specifically a requirement for the CI service. Also, if the build is going to fail before this point due to problems with tests or problems _building_ the docs, it's nice not to have wasted time installing doctr until you actually are ready to use it.